assets/js/components/DashboardViewIndicator.stories.js

```diff
@@ -59,29 +74,29 @@ export default {
 	title: 'Components/DashboardViewIndicator',
 	component: DashboardViewIndicator,
 	decorators: [
-		( Story, { args } ) => {
-			const registry = createTestRegistry();
-			provideSiteInfo( registry );
-			provideModules( registry, [
-				{
-					slug: 'analytics',
-					active: true,
-					connected: true,
-				},
-				{
-					slug: 'analytics-4',
-					active: true,
-					connected: true,
-				},
-			] );
-
-			args.setupRegistry?.( registry );
+		( Story ) => {
+			const setupRegistry = ( registry ) => {
+				provideSiteInfo( registry );
+				provideModules( registry, [
+					{
+						slug: 'analytics',
+						active: true,
+						connected: true,
+					},
+					{
+						slug: 'analytics-4',
+						active: true,
+						connected: true,
+					},
+				] );
```

Comment:
You should be able to restore the `args.setupRegistry()` callbacks and invoke it here, rather than creating additional decorators for each function and re-rendering `WithRegistrySetup` -

```suggestion
				] );
				
				args.setupRegistry?.( registry );
```

Reply:
Updated.